includes/definitions/discovery/eatonupsm2.yaml

```diff
@@ -13,8 +13,26 @@ modules:
             data:
                 -
                     oid: xupsEnvironment
-                    value: xupsEnvAmbientTemp
```

Comment:
Hi @dagbdagb 
Why did you remove this sensor. Are you sure it cannot be of any use for anybody ? Removing it here will remove it for each and every LibreNMS user after this PR is merged. In doubt, I would keep it, and also add you xupsEnvRemoteTemp,  so you end up with both.

Reply:
> Hi @dagbdagb
> Why did you remove this sensor. Are you sure it cannot be of any use for anybody ? Removing it here will remove it for each and every LibreNMS user after this PR is merged. In doubt, I would keep it, and also add you xupsEnvRemoteTemp, so you end up with both.

You are right of course. The current major release does not expose the onboard temperature sensor. But the previous release did, so we need to keep this. Improved version coming up. 

Thank you for reviewing.